charts/wasmcloud-platform/values.yaml

```diff
@@ -0,0 +1,86 @@
+# # Global values that apply to all subcharts
+# global:
+
+fullnameOverride: "wasmcloud-platform"
+nameOverride: "wasmcloud-platform"
+
+nats:
+  # Whether to install the nats chart (https://nats-io.github.io/k8s/helm/charts/nats)
+  enabled: true
+  fullnameOverride: "nats"
+  nameOverride: "nats"
+  # Source: https://github.com/wasmCloud/wasmcloud-operator/blob/main/examples/quickstart/nats-values.yaml
+  config:
+    cluster:
+      enabled: true
+      replicas: 3
+    leafnodes:
+      enabled: true
+    websocket:
+      enabled: true
+      port: 4223
+    jetstream:
+      enabled: true
+      fileStore:
+        pvc:
+          size: 10Gi
+      merge:
+        domain: default
+  # # Quickstart configuration
+  # statefulSet:
+  #   name: nats
+  # service:
+  #   name: nats
+  # headlessService:
+  #   name: nats-headless
+
+wadm:
+  # Whether to install the wadm chart (oci://ghcr.io/wasmcloud/charts/wadm)
+  enabled: true
+  fullnameOverride: "wadm"
+  nameOverride: "wadm"
+  # Source: https://github.com/wasmCloud/wasmcloud-operator/blob/main/examples/quickstart/wadm-values.yaml
+  wadm:
+    image:
+      tag: v0.14.0
```

Comment:
with [`v0.15.0` out now](https://github.com/wasmCloud/wadm/releases/tag/v0.15.0), I think we'll want to bump this up to that

Reply:
Updated